model.py

```diff
@@ -348,7 +349,8 @@ class User(orm.Document):
               ('username', orm.StringField(required=True)),
               ('salt', orm.StringField(required=True)),
               ('hashed_password', orm.StringField(required=True)),
-              ('admin_regions', orm.ListField(orm.StringField()))]
+              ('admin_regions', orm.ListField(orm.StringField())),
+              ('admin_level', orm.StringField(required=True, default='REGION'))]
```

Comment:
so, take a look at line 129 to see how it's done for tournament source_types (challonge, tio, smashgg).

basically, you should do something like
```
              ('admin_level', orm.StringField(
                  required=True,
                  validators=[orm.validate_choices(ADMIN_LEVEL_CHOICES)]))
```

and define a list/tuple ADMIN_LEVEL_CHOICES somewhere.